specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,264 @@
+# Background
+
+We are exposing an event that will fire when an attempt to launch a registered protocol is made. The host will be given the option to cancel the launch, handle the popup dialog to hide the dialog, as well as revoke permissions that have been previously given to the origin launching the protocol. 
+
+# Description
+
+This event will fire before the registered protocol launch occurs. Currently a popup dialog is displayed in which the user can click `Open` or `Cancel`. If the request is made from a trustworthy origin a checkmark box will be displayed that will allow the user to always allow this registered protocol from this origin. The `NavigationStarting`, `NavigationCompleted`, `SourceChanged`, `ContentLoading`, and `HistoryChanged` events will not fire when a request is made to launch a registered protocol. 
+
+There are two events associated with the registered protocol launch - one for the main frame, and one for non-main frame(s). In the case in which the launch request is made from a non-main frame, the frame will raise a `LaunchingRegisteredProtocol` event as well as `CoreWebView2.FrameLaunchingRegisteredProtocol` event. 
```

Comment:
There is no FrameLaunchingRegisteredProtocol event. Its also mentioned in comments later on.

Reply:
Just clarifying that you're referring to it should be `CoreWebView2.LaunchingRegisteredProtocol`?

Reply:
I mean you list two events here but there is only one event defined. The frame version of the event is not defined in this document. Although you do refer to it in some of the text and there is a ICoreWebView2FrameLaunchingRegisteredProtocolEventHandler interface defined.

Reply:
Oh right that was unclear sorry, only one event that fires off the corewebview2 and frame interface.